docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -516,9 +516,15 @@ policies:
     #        a very low alert volume or your upstream notification system performs
     #        its own grouping.
     group_by: ['...']
-    # <list> a list of matchers that an alert has to fulfill to match the node
```

Comment:
Hey Alex, looks good-just one thing-you are referring to an alert rule, right? in which case we should be specific and use that terminology to be clear.